quiz.js

```diff
@@ -1,5 +1,8 @@
 // Quiz: An Introduction to MongoDB
 
+// HI MAX/ INSTRUCTOR! I have checked this over and think it is right, but I cannot seem to find the houses in the mongo database. Please let me know in the grading if you figure out my error!
+
```

Comment:
Just test-ran your code, and I didn't have any trouble showing the contents of houses. Did you remember to run `use westeros` when you opened up Mongo afterwards?